vembrane/ann_types.py

```diff
@@ -68,6 +68,8 @@ def type_ann(
         new_key, value_func = KNOWN_ANN_TYPE_MAP.get(key, (key, try_auto_type))
         return new_key, value_func(value)
     else:
+        if key == "CLIN_SIG":
```

Comment:
I think the correct solution is to add an na_func to the type map. So if value evaluates to false, it can be called instead of value_func.

Reply:
Let us turn the type maps into a class, which wraps the dict and has a default na_func that returns na, but can be overwritten by field.

Reply:
good idea!